sched/Kconfig

```diff
@@ -1806,17 +1806,24 @@ endmenu # Work Queue Support
 
 menu "Stack and heap information"
 
+config ARCH_STACKSIZE_ADJUSTMENT
+	int "The adjustment of stack size for different arch"
+	default 65536 if ARCH_SIM
+	default 0 if !ARCH_SIM
+	---help---
+		The adjustment of stack size for different arch. When the task is
+		created, the stack size expands automatically depending on the arch.
```

Comment:
The adjustment of stack size for different arch. When the task is created, the stack size is increased by this amount.

Reply:
Done